### PR TITLE
Fetch N number of records in M milliseconds timeout before streaming the records

### PR DESCRIPTION
**Pull request by mertyildiran**
With this PR, the socket data streamer does a `basenine.Fetch` call with 50 limit and 3000 milliseconds timeout that's ordered by the front-end. The front-end orders this fetch to the WebSocket connection only in case of first ever WebSocket connection establish upon page load or querying. In the subsequent connections, front-end does not order this fetch.

### CODE REVIEW

**Comment by gadotroee** [comment]
I tried to run it locally and it seems to be ok, but I don't see the order logic in frontend.
and also it is giving be the 50 entries in the incorrect order

Adding a short screen recording to explain. 


https://user-images.githubusercontent.com/55343099/166652172-f94328ff-c077-4af1-acbf-56d6cdc690db.mov

**Comment by mertyildiran** [comment]
> I tried to run it locally and it seems to be ok, but I don't see the order logic in frontend. and also it is giving be the 50 entries in the incorrect order
> 
> Adding a short screen recording to explain.
> 
>  50-entries.mp4

@gadotroee you're right. I think I have it fixed with https://github.com/up9inc/mizu/commit/0adfb450547ef09beb454742e03588c15d366f5f Please confirm.

**Comment by gadotroee** [comment]
> > I tried to run it locally and it seems to be ok, but I don't see the order logic in frontend. and also it is giving be the 50 entries in the incorrect order
> > Adding a short screen recording to explain.
> > 50-entries.mp4
> 
> @gadotroee you're right. I think I have it fixed with [0adfb45](https://github.com/up9inc/mizu/commit/0adfb450547ef09beb454742e03588c15d366f5f) Please confirm.

I'm checking it now, can we just make an empty commit with #run_acceptance_tests message and see that if the changes here are causing any issue in the tests. if so ti understand what change do we need to make in the tests to support the changes that made here? 
(I can do it if you want but meanwhile when I'm testing the branch it can run..)

**Comment by mertyildiran** [comment]
@gadotroee I have realized that, without a change in the Basenine API, it cannot meet the exact feature specifications. So, I have done it with https://github.com/up9inc/mizu/commit/715403c2caefffc45f9c90f9e4f17031642bd2b8

**Comment by gadotroee** [comment]
> @gadotroee I have realized that, without a change in the Basenine API, it cannot meet the exact feature specifications. So, I have done it with [715403c](https://github.com/up9inc/mizu/commit/715403c2caefffc45f9c90f9e4f17031642bd2b8)

@mertyildiran 
Sorry about the delay. 
I have tried the new branch to see how it is working (before going over the code), and not sure. what exactly was the requirement that asked and didn't work before this change.

Actually I thought (from first look and simple check) that the changes before were ok but now when I tried this new changes I'm getting weird numbers in the bottom information.

(this is happening when I only started mizu and then after refresh go up to get "old" entries)
so in the first check it looks. like it is not working (which didn't happen before the change)

image

**Comment by mertyildiran** [comment]
> > @gadotroee I have realized that, without a change in the Basenine API, it cannot meet the exact feature specifications. So, I have done it with [715403c](https://github.com/up9inc/mizu/commit/715403c2caefffc45f9c90f9e4f17031642bd2b8)
> 
> @mertyildiran Sorry about the delay. I have tried the new branch to see how it is working (before going over the code), and not sure. what exactly was the requirement that asked and didn't work before this change.
> 
> Actually I thought (from first look and simple check) that the changes before were ok but now when I tried this new changes I'm getting weird numbers in the bottom information.
> 
> (this is happening when I only started mizu and then after refresh go up to get "old" entries) so in the first check it looks. like it is not working (which didn't happen before the change)
> 
> image

@gadotroee fixed with https://github.com/up9inc/mizu/commit/be73e876276fa2e05ab76d27af728d5ee0909ced Please check it again.

File: ui-common/src/components/TrafficViewer/TrafficViewer.tsx [hunk 1]
```diff
@@ -157,12 +153,18 @@ export const TrafficViewer: React.FC<TrafficViewerProps> = ({
     }
   }
 
-  const sendQueryWhenWsOpen = (query) => {
+  const sendQueryWhenWsOpen = (leftOff: string, query: string, _fetch: number) => {
```
[the remark this answers is not in the document]
Comment (mertyildiran) [reply ui-common/src/components/TrafficViewer/TrafficViewer.tsx]: @gadotroee fixed with https://github.com/up9inc/mizu/commit/4f9bf6e3ed41e344c2527a1b16cf99a6c2c27389

File: ui-common/src/components/TrafficViewer/TrafficViewer.tsx [hunk 2]
```diff
@@ -157,12 +153,18 @@ export const TrafficViewer: React.FC<TrafficViewerProps> = ({
...
     setTimeout(() => {
       if (ws?.current?.readyState === WebSocket.OPEN) {
-        ws.current.send(JSON.stringify({"query": query, "enableFullEntries": false}));
+        ws.current.send(JSON.stringify({
+          "leftOff": leftOff,
+          "query": query,
+          "enableFullEntries": false,
+          "fetch": _fetch,
+          "timeoutMs": 3000
```
[the remark this answers is not in the document]
Comment (mertyildiran) [reply ui-common/src/components/TrafficViewer/TrafficViewer.tsx]: @gadotroee fixed with https://github.com/up9inc/mizu/commit/dccdf5e4e3e9467a03b09e82605dcbfaec9d6e50

**Review by gadotroee — APPROVED** [review]

**Pull request merged**